addons/repair/functions/fnc_getHitPointsToIgnore.sqf

```diff
@@ -123,7 +123,7 @@ private _processedSelections = [];
         continue
     };
 
-    if ((_hitpointGroups findIf {(_x select 1) == _hitpoint}) != -1) then { // skip child hitpoints
+    if ((_hitpointGroups findIf {((_x select 1) findIf {_x == _hitpoint}) != -1}) != -1) then { // skip child hitpoints
```

Comment:
At this point this should be split into multiple lines. This is getting seriously unreadable 😅

Reply:
```sqf
#define ANY(set,condition) (set findIf {condition} != -1)

if ANY(_hitpointGroups, ANY(_x select 1, _x == _hitpoint))
```